README.md

```diff
@@ -83,6 +83,10 @@ Our datasets are here: https://git.uwaterloo.ca/r33tang/daam-data
 ## See Also
 - [1littlecoder's video](https://www.youtube.com/watch?v=J2WtkA1Xfew) for a code demonstration and Colab notebook of an older version of DAAM.
 
+- [Easiest way to use DAAM script tutorial](https://www.youtube.com/watch?v=XiKyEKJrTLQ)
+
+[![image.png](https://s3.amazonaws.com/moonup/production/uploads/1675628788246-6345bd89fe134dfd7a0dba40.png)](https://www.youtube.com/watch?v=XiKyEKJrTLQ)
```

Comment:
Please remove this image.

Reply:
@daemon thank you so much. removed the image as requested. sorry for being late